rfcs/0xxx-unified-sharing.md

```diff
@@ -0,0 +1,71 @@
+# Unified Sharing
+
+Support a consistent, flexible model of "personal" vs "shared" item visibility in Signals, Queries, and Dashboards.
+
+## Motivation
+
+Seq 4.2 has two distinct models for sharing common items in the main events/dashboards screens:
+
+| Signals & Queries | Dashboards |
+| --- | --- |
+| Shared only | Shared or personal |
+| Shared by default | Personal by default |
+| Shared items openly editable | Shared items admin-only |
+| "Restricted" editing option | - |
+| Explicit visibility selection | All items visible |
+
+This leaves gaps that have been noted by users:
+
+ * It's not possible to "hide" items from a large dashboard list
+ * It's not possible for non-admin users to share dashboards, or edit shared dashboards
+ * New signals created by one user don't appear/aren't discoverable by other users
+ * Throwaway/temporary "personal" signals clutter the global shared signal list
+
+This RFC seeks to unify these two sharing models, so that one well-aligned mental model applies for all shared resources dealt with day-to-day in Seq.
+
+## Detailed design
+
+The following guiding principles have been established:
+
+ 1. Sharing is an explicit choice. It should be easy and comfortable to work in Seq without worrying about cluttering up shared resources. Creating a new shared resource, or modifying an existing one, should be an obvious and deliberate action.
+ 1. Collaboration is the default. While administrative users may need to be able to restrict editing of sensitive signals and dashboards, this is the exception rather than the rule. Users should be free to collaborate and improve these things by default.
+ 1. [Labels]() and [Signal Groups]() are the primary means of dealing with clutter. It's unwieldy to rely on "hiding" items to keep lists clean, as personal roles/focus may change regularly.
+
+Applying these, the model proposed by this RFC is:
+
+ * All items can be marked with an "owner", in which case they will be regarded as "personal"
+ * All newly-create items will initially be owned by their creator (except in API-driven/automation scenarios)
+ * Any user can make a "personal" item into a "shared" item
+ * Administrative users can mark shared items as "restricted", in which case they can only be modified by administrative users
+ * Any user can edit, remove, or make personal any unrestricted "shared" item
+ * Users will by default see all of their personal items, and all shared items, in each category
+ * Shared items can be "hidden", which corresponds with the current "Remove" function on signals
```

Comment:
Can shared items also be deleted?

Reply:
I've updated the wording of the item two above this, to make clear that all items can be deleted (previously this incorrectly used the term "remove")